src/ultra/test.clj

```diff
@@ -43,6 +44,19 @@
         (diff/print-expected actual expected))
       (logic/maybe-print-values event)))
 
+(defmethod report :error [m]
```

Comment:
This needs to go within the body of `activate!`, above -- otherwise it'll end up being active even when Ultra features are disabled.